### ISSUE TITLE
How to use is_process_running ?

### ISSUE THREAD

**Issue by rockyzhz** [pywinauto/pywinauto#734]
Document says :
        Check that process is running.
        Can be called before start/connect.
        Return True if process is running otherwise - False.

So, my problem is :

before I start/connect a process, how to test it?

thanks!

**Comment by airelil** [comment]
@rockyzhz, I think you are right, the doc is misleading. The meaning of the comment was that calling the method before start/connect won't crash the script. The purpose of the method is mostly to verify that automated process still exists. We need to update the doc. Also a small example for clarity:
```python
import pywinauto

app = pywinauto.Application()
# Will return False
print(app.is_process_running())

# Now after connecting to a running application
# we'll get True
app.connect(path="vim.exe")
print(app.is_process_running())

```

**Comment by rockyzhz** [comment]
> @rockyzhz, I think you are right, the doc is misleading. The meaning of the comment was that calling the method before start/connect won't crash the script. The purpose of the method is mostly to verify that automated process still exists. We need to update the doc. Also a small example for clarity:
> 
> ```python
> import pywinauto
> 
> app = pywinauto.Application()
> # Will return False
> print(app.is_process_running())
> 
> # Now after connecting to a running application
> # we'll get True
> app.connect(path="vim.exe")
> print(app.is_process_running())
> ```

So, according to my understanding, if I want to test whether an application is running or not,
so that I can choose the method between 'start' and 'connect' to use, a reasonable solution is:
```
import pywinauto

process_is_running = True
try:
    app = pywinauto.Application().connect(path="vim.exe")
except pywinauto.application.ProcessNotFoundError:
    process_is_running = False

print(process_is_running)
```

**Comment by airelil** [comment]
Right. It looks reasonable for me too: if you get an exception on ```connect``` call you should fall back to ```start```.  Later on, during the run of your automation, you can use ```is_process_running``` if you want to verify that the automated application is still alive.
